CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
```

Comment:
We probably want to start indicating breaking changes as more users start to use our instrumentations so please rename this to `### Breaking Changes`.